src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,27 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGTH_PASSWORD = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new NullPointerException("User is null");
+        }
+        if (user.getLogin() == null) {
+            throw new NullPointerException("Login is null");
+        }
```

Comment:
```suggestion
        if (user == null) {
            throw new RuntimeException("User is null");
        }
        if (user.getLogin() == null) {
            throw new RuntimeException("Login is null");
        }
```

it's bad practice to throw NPE manually https://stackoverflow.com/questions/3322638/is-it-okay-to-throw-nullpointerexception-programmatically